exonum/src/api/node/public/explorer.rs

```diff
@@ -400,6 +470,18 @@ impl ExplorerApi {
                 let blockchain = self.blockchain.clone();
                 move |query| Self::block(blockchain.snapshot().as_ref(), query)
             })
+            .endpoint("v1/call_status/transaction", {
```

Comment:
Without the authenticity proof, this endpoint is just a worse version of the `transaction` endpoint. I think we should get this PR merged already, so this can be solved separately.